cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -65,7 +65,8 @@ class TLM:
             with "best" also returning more reliable trustworthiness scores than "high".
             The "medium" and "low" presets return standard LLM responses along with associated trustworthiness scores,
             with "medium" producing more reliable trustworthiness scores than low.
-            The "base" preset will not return any trustworthiness score, just a standard LLM response, and is similar to directly using your favorite LLM API.
+            The "base" preset will provide a standard LLM response and a trustworthiness score in the lowest possible latency.
```

Comment:
we should also indicating that if `"use_self_reflection": False` then it will return None since we only use the self reflection for the "base" preset scoring

Reply:
I added a note here instead: https://github.com/cleanlab/cleanlab-studio/pull/302/commits/582e500198c367c6a4655484cefacd759ee8d60f